rollup.config.js

```diff
@@ -38,12 +38,7 @@ module.exports = functions.map(fn => ({
       include: ['dist/**/*.{ts,tsx}'],
     }),
     resolve(),
-    commonjs({
-      namedExports: {
-        'node_modules/lodash/lodash.js': ['isObject', 'trimStart', 'cloneDeep', 'get', 'set'],
-        'node_modules/@stoplight/types/dist/index.js': ['DiagnosticSeverity'],
-      },
-    }),
+    commonjs(),
```

Comment:
Addresses the following deprecation warning.
![image](https://user-images.githubusercontent.com/9273484/83362053-9513ae80-a38e-11ea-8a89-c6e6f66ea104.png)